File: articles/conflict-resolution-organizational-systems/cpp/conflict_resolution_score.cpp
/*
Conflict Resolution in Organizational Systems
Simple C++ scoring demonstration.

Responsible-use scope:
Synthetic-data research, methods demonstration, institutional learning,
and reproducible workflows only. Not for employment, cultural-loyalty scoring,
conflict-proneness scoring, collaboration scoring, productivity ranking, or
individual worker decisions.
*/

#include <iostream>

struct ConflictResolutionProfile {
    double trust_level;
    double communication_clarity;
    double psychological_safety;
    double procedural_fairness;
    double integrative_skill;
    double blame_intensity;
    double power_asymmetry;
    double resource_strain;
    double mediation_access;
};

double constructive_conflict_capacity_score(const ConflictResolutionProfile& p) {
    return
        0.16 * p.trust_level +
        0.15 * p.communication_clarity +
        0.14 * p.psychological_safety +
        0.15 * p.procedural_fairness +
        0.12 * p.integrative_skill +
        0.10 * p.mediation_access -
        0.10 * p.blame_intensity -
        0.09 * p.power_asymmetry -
        0.07 * p.resource_strain;
}

int main() {
    ConflictResolutionProfile profile{82, 80, 81, 83, 76, 18, 22, 45, 84};
    std::cout << "Synthetic constructive conflict capacity score: "
              << constructive_conflict_capacity_score(profile) << "\n";
    std::cout << "Responsible-use reminder: institutional learning only.\n";
    return 0;
}

File: articles/conflict-resolution-organizational-systems/fortran/conflict_resolution_score.f90
! Conflict Resolution in Organizational Systems
! Simple Fortran scoring demonstration.
!
! Responsible-use scope:
! Synthetic-data research, methods demonstration, institutional learning,
! and reproducible workflows only. Not for employment, conflict-proneness
! scoring, collaboration scoring, productivity ranking, or individual decisions.

program conflict_resolution_score_program
  implicit none
  real :: trust_level, communication_clarity, psychological_safety
  real :: procedural_fairness, integrative_skill, blame_intensity
  real :: power_asymmetry, resource_strain, mediation_access, score

  trust_level = 82.0
  communication_clarity = 80.0
  psychological_safety = 81.0
  procedural_fairness = 83.0
  integrative_skill = 76.0
  blame_intensity = 18.0
  power_asymmetry = 22.0
  resource_strain = 45.0
  mediation_access = 84.0

  score = 0.16 * trust_level + 0.15 * communication_clarity + &
          0.14 * psychological_safety + 0.15 * procedural_fairness + &
          0.12 * integrative_skill + 0.10 * mediation_access - &
          0.10 * blame_intensity - 0.09 * power_asymmetry - &
          0.07 * resource_strain

  print *, "Synthetic constructive conflict capacity score:", score
  print *, "Responsible-use reminder: institutional learning only."
end program conflict_resolution_score_program
